### DISCUSSION TITLE
Add HTML indexing capabilities

### DISCUSSION THREAD

**Discussion by martonx** [meilisearch/product#505] [Feedback & Feature Proposal]
I had a discussion in Q&A section. I have a feature proposal based on this discussion: https://github.com/meilisearch/meilisearch/discussions/2650#discussioncomment-3308931

I hope you'll like the idea to index html content. So the base idea is to send the content html, and the indexer would index only the scraped, cleaned content, not the html elements, classes, tags etc.

**Comment by gmourier** [comment]
Hi @martonx

We already discussed that, and the current statement is that we are not dealing with user data cleaning. However, this will probably be considered in the future when we finish our essential work in progress; it could be in the shape of an indexing pipeline welcoming multiple routines like trimming, HTML sanitization, translation from a third party, etc...

For now, the workaround is to have two fields: one in HTML format for displaying the search result and a HTML sanitized version for searching (the latter must be specified in the searchable attributes).

**Comment by ManyTheFish** [comment]
Hey @martonx! This is related to https://github.com/meilisearch/product/discussions/474, don't hesitate to upvote it and add some explanation of your use case!

**Reply by gmourier** [reply]
Thanks @ManyTheFish and @martonx, I'm locking that one in favor of #474
